database/parameters.m

```diff
@@ -170,7 +170,7 @@
 if (seek == "HREV1")  
     % engine locations
     retParams.X_GM = 2.244;                                                             % engine location from nose tip                     [m]
-    retParams.X_CG_full = 0.2745;                                                       % location of cg of engine from X_GM                [m]
+    retParams.X_CG_full = 0.2745;                                                       % location of cg of engine from X_GM ( to the tank tip) [m]                [m]
```

Comment:
So in this case X_GM is the length from tip of nosecone to tip of tank (ie. tip of tank is what we are considering to be the engine start point) - nothing to really change, but we could make the commenting a little clearer in the future

Reply:
Thanks! I know you said nothing to change really, but I made a much clearer comment there.